src/EditorFeatures/CSharpTest2/EmbeddedLanguages/Json/CSharpJsonParserTests_NstTests.cs

```diff
@@ -4724,12 +4724,12 @@ public void n_string_escaped_emoji_json()
             """,
     """
     <Diagnostics>
-      <Diagnostic Message="Invalid escape sequence" Start="13" Length="3" />
+      <Diagnostic Message="Invalid escape sequence" Start="13" Length="2" />
     </Diagnostics>
     """,
     """
     <Diagnostics>
-      <Diagnostic Message="Invalid escape sequence" Start="13" Length="3" />
+      <Diagnostic Message="Invalid escape sequence" Start="13" Length="2" />
```

Comment:
acceptable behavior change `\🌀` is three characters the backslash, then a high surrogate then a low surrogate.  The json parser is simply saying that the character that follows the `\` is not legal.  so it only marks those two chars, instead o the three from before when we merged the surrogates into one rune.